src/test/unit/__snapshots__/profile-conversion.test.js.snap

```diff
@@ -372,29 +372,29 @@ Object {
         ],
         "fileName": Array [
           null,
-          3,
+          2,
```

Comment:
now we don't use the stringTable to generate an index for the address, so some string indexes are decremented.